src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -378,4 +385,109 @@ public InputStream body() throws IOException {
         }
     }
 
+    /**
+     * Fake decorator.
+     * @since 0.16
+     */
+    final class Fake implements RqMultipart {
+
+        /**
+         * Fake multipart request.
+         */
+        private final RqMultipart fake;
+
+        /**
+         * Fake ctor.
+         * @param req Fake request header holder
+         * @param dispositions Fake request body parts
+         * @throws IOException If fails
+         */
+        public Fake(final Request req, final String... dispositions)
```

Comment:
I think this is wrong. this would be better: `Fake(final Request req, final Request... dispositions)`